include/lsst/ctrl/events/EventTransmitter.h

```diff
@@ -41,41 +41,38 @@
 #include <stdlib.h>
 #include <iostream>
 
-#include "lsst/ctrl/events/Event.h"
-#include "lsst/utils/Utils.h"
-#include "lsst/daf/base/PropertySet.h"
-#include "lsst/ctrl/events/EventBroker.h"
+#include "lsst/ctrl/events/Transmitter.h"
 
 using lsst::daf::base::PropertySet;
 
 namespace lsst {
 namespace ctrl {
-namespace events { 
+namespace events {
 
 /**
  * @class EventTransmitter
  * @brief Transmit events to the event bus
  */
 
-class EventTransmitter
+class EventTransmitter : public Transmitter
 {
 public:
+
+    typedef boost::shared_ptr<EventTransmitter> Ptr;
+
     /** 
-     * @brief Transmits events to the specified host and topic
+     * @brief Transmits events to the specified host and destination
      *
      * @param hostName the machine hosting the message broker
-     * @param topicName the topic to transmit events to
+     * @param destinationName the topic to transmit events to
```

Comment:
I'm a bit confused by the interchangeable use of topic and destination. Should the help for destinationName really be using "topic"? Perhaps a few more words would help?